config/environments/production.rb

```diff
@@ -81,7 +81,11 @@
   # Use default logging formatter so that PID and timestamp are not suppressed.
   config.log_formatter = ::Logger::Formatter.new
 
-  config.logstasher.enabled = true
-  config.logstasher.logger = Logger.new("#{Rails.root}/log/#{Rails.env}.json.log")
-  config.logstasher.suppress_app_log = true
+  if ENV["DYNO"].present?
```

Comment:
I've created a PR for this: https://github.com/alphagov/government-frontend/pull/80